docs/field-details.asciidoc

```diff
@@ -855,6 +855,51 @@ example: `co.uk`
 // ===============================================================
 
 
+|=====
+
+[[ecs-dll]]
+=== DLL Fields
+
+These fields contain information about loaded process modules.
+
+Many operating systems call process modules different names, but this field set refers all of the following: * Dynamic-link library (`.dll`) commonly used in Windows * Shared Object (`.so`) commonly used in Unix-like operating systems * Dynamic library (`.dylib`) commonly used in macOS
```

Comment:
i guess the newlines didn't get preserved. maybe I should've switched YAMLs `>` to `|-`?

Reply:
Huh, I didn't know about `|-`, nice fix :-)